projects/angular/src/forms/styles/_textarea.clarity.scss

```
@@ -7,14 +7,15 @@
 @use '../../utils/mixins';
 @use 'mixins.forms' as forms-mixins;
 @use 'variables.forms' as forms-variables;
+@use '../../layout/density' as density;
 @use '@cds/core/tokens/tokens.scss';
 
 @include mixins.exports('forms.textarea') {
   // @TODO Fix selectors when old forms are removed to allow better defaults
   .clr-textarea-wrapper {
     white-space: nowrap;
     display: flex;
-    margin-top: tokens.$cds-global-space-5;
+    margin-top: density.$clr-base-gap-xs;
```

The default margin was 8px and with this tokens it gets 4px so it should use bigger size.
```suggestion
    margin-top: density.$clr-base-gap-s;
```